app/helpers/loss_report_helper.rb

```diff
@@ -0,0 +1,189 @@
+module LossReportHelper
+
+  def loss_report_output(compared_collections: compared_collections, hide_items: true, display: 'text')
```

Comment:
Sorry if I was unclear. You are not providing a default value here for compared_collections (I think the "default value" here actually doesn't mean anything and if you tried calling the method without passing in any arguments it would throw an error), so this should look like `def loss_report_output(compared_collections:, hide_items: true, display: 'text')`

Reply:
okay I see, Is there any benefit to using symbols for plain arguments?

Reply:
Consistency with your other arguments and argument order agnosticicity (agnosticness?)